src/custom/pages/Faq/index.tsx

```diff
@@ -326,20 +349,116 @@ export default function Faq() {
-            time a solver submits a successful batch settlement solution, the protocol rewards them with GNO. 
-            Anyone can become a solver, although, in order to become one, there are certain requirements:
+            time a solver submits a successful batch settlement solution, the protocol rewards them with GNO. Anyone can
+            become a solver, although, in order to become one, there are certain requirements:
           </p>
           <ol>
             <li>To become a solver, an Ethereum address needs to deposit a bond of GNO tokens to GnosisDAO.</li>
             <li>
-              Once the GNO tokens have been staked (locked up), GnosisDAO has to vote to approve or reject the
-              Ethereum address that will identify the solver.
+              Once the GNO tokens have been staked (locked up), GnosisDAO has to vote to approve or reject the Ethereum
+              address that will identify the solver.
             </li>
             <li>
               Additionally, a solver must have the technical knowledge to create the appropriate batch settlement
               solutions, or take the risk of being slashed by the GnosisDAO for wrongdoing.
             </li>
           </ol>
+
+          <h3 id="what-interactions-can-i-encounter-when-using-Cowswap">
+            What interactions can I encounter when using CowSwap?
+          </h3>
+
+          <p>
+            <strong>Internal CowSwap Operations</strong>
+          </p>
+          <div id="table-container">
+            <table>
+              <thead>
+                <tr>
+                  <th>Action</th>
+                  <th>Description</th>
+                  <th>Costs</th>
+                  <th>Action Performed</th>
+                  <th>Pay for gas?</th>
+                  <th>What do I need to pay the gas costs with?</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Approve token for trading</td>
+                  <td>
+                    Required step for being able to sell a token.Only needs to be done once. Afterward, you will be able
+                    to trade the token using gasless transactions!
+                  </td>
+                  <td>Regular Ethereum Tx.</td>
+                  <td>Set the token allowance for the Allowance Manager</td>
+                  <td>Yes</td>
+                  <td>ETH</td>
+                </tr>
+                <tr>
+                  <td>Signing an Order</td>
+                  <td>
+                    Signature of a gasless off-chain order. You define your limit price and expiration date. The order
+                    will try to be executed using MEV protection against different on-chain liquidity sources or other
+                    CowSwap users trading in the same block.
+                  </td>
+                  <td>Free</td>
+                  <td>None</td>
```

Comment:
Well, I'd argue that the action is signing the order/message on user's wallet

Reply:
But, what would you change the text to @alfetopito ? Something like: 
 Signature of a gasless off-chain order on the users' wallet. You define your limit price and expiration date. The order will try to be executed using MEV protection against different on-chain liquidity sources or other                    CowSwap users trading in the same block.

Reply:
Not talking about updating the base text. My suggestion is to add an `action` instead of leaving it `None`